Create a pull_request_template.md (#203)
exasim-project/NeoFOAM: .github/pull_request_template.md

# Motivation

> Please add a short description what this PR provides or why it is needed.

Reviewer: ```suggestion
WHAT is it, WHY it is needed. :kissing_heart: 
```

> # Motivation
> Please add a short description what this PR provides or why it is needed.

Author: ```suggestion
# Motivation
Add a sentence why it is needed?
```
